conformance/lib/build.gradle.kts

```diff
@@ -79,12 +79,4 @@ tasks {
         kotlinPackage.set("pbandk.conformance.pb")
         logLevel.set("debug")
     }
-
-    // DCE is now enabled by default in Kotlin 1.3.7x
-    // and it doesn't work well with commonJS modules
-    // Use of commonJs could be removed since default module is now UMD
-    // but would require some code change
-    val processDceJsKotlinJs by getting {
```

Comment:
`processDceJsKotlinJs` is unavailable when you use both JS compiler backends, so I did not have much of a say in keeping this option or not. I can try adding this to the build config: https://kotlinlang.org/docs/javascript-dce.html#disable-dce.

But anyway, the conformance tests pass without tinkering with DCE, so I am not sure we even need to do anything.

Reply:
If the conformance tests run fine then removing this is probably ok. It might be worth running the JS app in the examples directory as well just to make sure. The conformance tests run under nodejs, so they wouldn't have tested the Kotlin/JS browser output.

Reply:
Hmm, the conformance tests work fine for both legacy and IR compiler backend. But I have just tried the browser-js example and indeed some of the pbandk API is unavailable there for some reason (I assume because of DCE). I have tried adding https://kotlinlang.org/docs/javascript-dce.html#disable-dce to both runtime and the example itself, neither helped unfortunately. I will try to poke around some more in free time, maybe there is a more reliable way to disable DCE even if you cross-compile to both backends...

Reply:
Ok, I have managed to get `browser-js` example to work, but I had to upgrade the example to use Kotlin 1.4. It just refused to work on Kotlin 1.3 and I could not find the source of the problem...

In the process, I also had to patch one other problem with generating `orDefault` methods.

Reply:
Updating the examples to Kotlin 1.4 is great. That's something we needed to do anyways. We're only supporting Kotlin 1.3 on a best-effort basis in pbandk.